app/helpers/application_helper.rb

```diff
@@ -5,6 +5,10 @@ def application_name
     'Argo'
   end
 
+  def search_params
```

Comment:
Can this be on the component rather than in the helpers? If not, can it be in the BulkActionHelper?

Reply:
Have you considered using the blacklight method "search_state" instead?

Reply:
Didn't know that exited.  I moved this helper to the component and am using blacklight's search_state instead (in the component and in other places directly).